### ISSUE TITLE
Plugin not working in latest flake8 version

### ISSUE THREAD

**Issue by guigarfr** [globality-corp/flake8-logging-format#35]
flake8 has released a couple of version in the last days (5.0.0 and 5.0.1). When running flake8, it provides this error (fails with both flake8 versions):

```bash
pip install flake8==5.0.0
flake8 .
```

```
There was a critical error during execution of Flake8:
plugin code for `flake8-logging-format[logging-format]` does not match ^[A-Z]{1,3}[0-9]{0,3}$
```

**Comment by guigarfr** [comment]
https://github.com/globality-corp/flake8-logging-format/pull/33

**Comment by cpina** [comment]
Just found the same problem - thanks for reporting it @guigarfr

**Comment by christianbundy** [comment]
Looks like #33 was merged (thanks!), looking forward to upgrading once this is released to pypi.

**Comment by christianbundy** [comment]
@tonybajan-globality Anything we can do to help get this published to pypi?

**Comment by guigarfr** [comment]
https://github.com/globality-corp/flake8-logging-format/issues/42

**Comment by piercefreeman** [comment]
Thanks for the contributions all. We've fixed the build pipeline and cut 0.7.4 to pypi, which should take care of this error. I've confirmed in another dependent repo that this issue looks fixed on flake8 5.x.x.
